code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -120,6 +119,14 @@
 	last_healed = world.time + heal_interval
 
 /obj/effect/alien/resin/special/pylon/core/attack_alien(mob/living/carbon/Xenomorph/M)
+	if(!warn && world.time < HIVECORE_COOLDOWN_CUTOFF)
+		if((alert(M, "Are you sure that you want to destroy the hive core? (5 minute cooldown before you can build another one.)", , "Yes", "No") == "No"))
+			return XENO_NO_DELAY_ACTION
+		xeno_message(SPAN_XENOANNOUNCE("[M] is destroying the Hive Core!"), 3, linked_hive.hivenumber)
+		addtimer(VARSET_CALLBACK(src,warn, FALSE), 15 SECONDS)
```

Comment:
this... also does nothing since nowhere is warn set to true

Reply:
no, it checks if warn is false, though I will reverse that because its confusing